ynr/apps/moderation_queue/templates/moderation_queue/photo-upload-new.html

```diff
@@ -13,8 +13,13 @@ <h1>Upload a photo of {{ person.name }}</h1>
 
 {% block content %}
 
-{% include 'moderation_queue/_photo-policy.html' %}
-{% include 'moderation_queue/_photo-upload-image-form.html' with person_id=person.id %}
-{% include 'moderation_queue/_photo-upload-url-form.html' with person_id=person.id %}
-
+{% if not queued_images %}
```

Comment:
Change this to only show form if queued images exist for the current person.

Reply:
I think that is already the case, looking at what the `queued_images` var is in the view
https://github.com/DemocracyClub/yournextrepresentative/blob/7a1a61ed7db11965c5fc6ab6973fe749c874b300/ynr/apps/moderation_queue/views.py#L63